### PR TITLE
EOS-20730 : Symas replication issue

### PR DESCRIPTION
**Pull request by pranavPawar07**
Signed-off-by: root <root@ssc-vm-2382.colo.seagate.com>

### CODE REVIEW

**Comment by pranavPawar07** [comment]
S3 pre-merge : http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Component-Test/job/S3Server-PreMerge-Test/lastBuild/console
Mini prov: http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Mini-Provisioning/job/cortx-s3server/1087/console

**Comment by nileshgovande** [comment]
@pranavPawar07 
Please test 
1)3 node provisioning and ldap replication
2)single node mini provisioning
3)./init.sh for dev environment once changes are complete as the scripts are common

**Review by abhilekh268 — APPROVED** [review]

**Comment by nileshgovande** [comment]
test this please

**Comment by pranavPawar07** [comment]
> test this please

@nileshgovande in progress.

File: scripts/ldap/setup_ldap.sh [hunk 1]
```diff
@@ -96,8 +95,23 @@ rm -rf /etc/openldap/slapd.d/cn\=config/olcDatabase\=\{2\}mdb.ldif
 if [[ $forceclean == true ]]
 then
   rm -rf /var/lib/ldap/*
+  if rpm -q 'symas-openldap-clients' 
+  then
+    yum remove -y symas-openldap-clients
```
Comment (nileshgovande) [inline scripts/ldap/setup_ldap.sh:100]: These changes should not be merged to production (as openldap comes through RE third party framework) - Can be kept for testing

Comment (nileshgovande) [reply scripts/ldap/setup_ldap.sh:100]: Also - move these changes to init.sh allied ansible 
in provisioner environment, RE will handle the OpenLDAP installation

**Pull request closed without merging**